adserver/templates/adserver/base.html

```diff
@@ -73,12 +73,21 @@ <h6 class="text-muted">{{ advertiser }}</h6>
             <h6 class="text-muted">{{ publisher }}</h6>
 
             <ul class="nav flex-column mb-2">
+
               <li class="nav-item">
                 <a class="nav-link" href="{% url 'publisher_report' publisher.slug %}">
                   <span class="fa fa-bar-chart fa-fw mr-2 text-muted" aria-hidden="true"></span>
                   <span>{% trans 'Reports' %}</span>
                 </a>
               </li>
+
+              <li class="nav-item">
```

Comment:
This block should only appear if the publisher allows `unauthed_ad_decision`. Otherwise, it won't work because the JSONP request will be rejected.